experimental/packages/opentelemetry-sdk-node/test/sdk.test.ts

```diff
@@ -1095,6 +1101,218 @@ describe('Node SDK', () => {
       await sdk.shutdown();
     });
   });
+
+  describe('configuring metric provider from env', () => {
+    let stubLogger: Sinon.SinonStub;
+
+    beforeEach(() => {
+      stubLogger = Sinon.stub(diag, 'info');
+    });
+
+    afterEach(() => {
+      stubLogger.reset();
+      delete env.OTEL_METRICS_EXPORTER;
+      delete env.OTEL_EXPORTER_OTLP_METRICS_PROTOCOL;
+      delete env.OTEL_EXPORTER_METRICS_PROTOCOL;
+    });
+
+    it('should not register the provider if OTEL_METRICS_EXPORTER is not set', async () => {
+      const sdk = new NodeSDK();
+      sdk.start();
+      assert.ok(!(metrics.getMeterProvider() instanceof MeterProvider));
+      await sdk.shutdown();
+    });
+
+    it('should not register the provider if OTEL_METRICS_EXPORTER contains none', async () => {
+      env.OTEL_METRICS_EXPORTER = 'console,none';
+      const sdk = new NodeSDK();
+      sdk.start();
+      assert.ok(!(metrics.getMeterProvider() instanceof MeterProvider));
+      assert.strictEqual(
+        stubLogger.args[1][0],
+        'OTEL_METRICS_EXPORTER contains "none". Metric provider will not be initialized.'
+      );
+      await sdk.shutdown();
+    });
+
+    it('should use console with default interval and timeout', async () => {
+      env.OTEL_METRICS_EXPORTER = 'console';
+      const sdk = new NodeSDK();
+      sdk.start();
+      const meterProvider = metrics.getMeterProvider();
+      const sharedState = (meterProvider as any)['_sharedState'];
+      assert(
+        sharedState.metricCollectors[0]._metricReader._exporter instanceof
+          ConsoleMetricExporter
+      );
+      assert(
+        sharedState.metricCollectors[0]._metricReader._exportInterval === 60000
+      );
+      assert(
+        sharedState.metricCollectors[0]._metricReader._exportTimeout === 30000
+      );
+      await sdk.shutdown();
+    });
+
+    it('should use otlp with gRPC and default interval and timeout', async () => {
+      env.OTEL_METRICS_EXPORTER = 'otlp';
+      env.S = 'grpc';
+      const sdk = new NodeSDK();
+      sdk.start();
+      const meterProvider = metrics.getMeterProvider();
+      const sharedState = (meterProvider as any)['_sharedState'];
+      assert(
+        sharedState.metricCollectors[0]._metricReader._exporter instanceof
+          OTLPGrpcMetricExporter
+      );
+      assert(
+        sharedState.metricCollectors[0]._metricReader._exportInterval === 60000
+      );
```

Comment:
When checking for strict equality, you can use this
```suggestion
      assert.strictEqual(sharedState.metricCollectors[0]._metricReader._exportInterval, 60000);
```

this'll give more useful error messages if the test fails, please also align the other tests you added where possible.